packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -265,7 +265,12 @@ export const ReleaseDetailsLayout = ({
   };
 
   const handleRefresh = () => {
-    dispatch(releaseApi.util.invalidateTags([{ type: 'ReleaseAction', id: 'LIST' }]));
+    dispatch(
+      releaseApi.util.invalidateTags([
```

Comment:
is this type safe?

Reply:
It is. It checks `type` to make sure is one of the [tagTypes defined](https://github.com/strapi/strapi/blob/develop/packages/core/content-releases/admin/src/services/release.ts#L52) on `createApi`

Reply:
yeah, you can manually invalidate the cache even if it is not reccomended